es6-promises/main.js

```diff
@@ -0,0 +1,6 @@
+import takeAChance from './take-a-chance.js';
+
+const name = takeAChance('bug');
+
+name.then((val) => console.log(val));
+name.catch((val) => console.log(val));
```

Comment:
This doesn't work properly unless you use chaining:
`name.then().catch()`

The problem is rather technical, but what happens is you end up with 2 rejections, one of which never gets handled, so node complains. Line 1 in `take-a-chance.js` turns off this complaint. If you remove that line you'll see it. (I've been meaning to remove that line for a long time but keep forgetting. I just did it so the next cohort won't be misled.)